src/neo/SmartContract/ContractParametersContext.cs

```diff
@@ -62,6 +62,11 @@ public JObject ToJson()
         /// </summary>
         public readonly DataCache Snapshot;
 
+        /// <summary>
+        /// The magic number of the network.
+        /// </summary>
+        public readonly uint Magic;
```

Comment:
Network?

Reply:
I mean only modify this field.

Reply:
I know, but you don't think that it's better to call it network in other places?